orderbook/openapi.yml

```diff
@@ -265,6 +291,29 @@ components:
       allOf:
         - $ref: "#/components/schemas/OrderCreation"
         - $ref: "#/components/schemas/OrderMetaData"
+    Trade:
+      description: |
+        Trade data such as executed amounts, fees, order id and block number.
+      type: object
+      properties:
+        blockNumber:
+          description: "Block in which trade occurred"
+          type: integer
+        logIndex:
+          description: "Index in which transaction was included in block"
+          type: integer
+        orderUid:
+          description: "Unique ID of the order matched by this trade"
+          $ref: "#/components/schemas/UID"
+        sellAmount:
+          description: "Amount sold in this trade"
+          $ref: "#/components/schemas/BigUint"
+        buyAmount:
```

Comment:
should we specify they are in atoms?

Reply:
If we denote that here we should also change it in the orders API (where we have the cummulative amount). Maybe we can add unit as a parameter type later (not sure we want atoms and base unit amounts in the same response).

Reply:
okok